### ISSUE TITLE
Text spacing rule should not fail empty elements

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4232]
A recent update to ACT added test cases to the spacing not important rules where there is no text effected by the spacing. Axe-core should not fail such examples:

```html
<div style="letter-spacing: 0.1em !important;"></div>
```

```html
<div style="letter-spacing: 0.1em !important">
	<p style="letter-spacing: 0.2em !important;">
		The toy brought back fond memories of being lost in the rain forest.
	</p>
</div>
```

See: https://www.w3.org/WAI/standards-guidelines/act/rules/24afc2/proposed/
